pkg/compiler/codegen.go

```diff
@@ -301,6 +301,16 @@ func isInitFunc(decl *ast.FuncDecl) bool {
 		decl.Type.Results.NumFields() == 0
 }
 
+func (c *codegen) isDeployFunc(decl *ast.FuncDecl, pkg *types.Package) bool {
+	if c.mainPkg.Pkg != pkg || decl.Name.Name != "_deploy" || decl.Recv != nil ||
```

Comment:
Why only `mainPkg`? I thought we might have it like `init()`.

Reply:
I don't like the idea of implicit initialization which also can't be overrided. May be we can name this method `Deploy` to be able to call it explicitly, if we import some other contract?

Reply:
I don't see a use case for `Deploy`. It's supposed to be called only once, if one really wants to export that, it could be done easily (export `Deploy` and call it from `_deploy`), but for what purpose if it's going to be called from `_deploy` anyway? I think it'd be more useful for us to have `_deploy()` behave like `init()` in that any package can define it for itself and then `_deploy` for the whole contract would go through all of them.